src/tests/evidence_scoring.py

```
@@ -0,0 +1,116 @@
+
+    return {
+        'stance_match': stance_is_match,
+        'max_content_similarity': max_sim
+    }
+
+def main():
+    stance_correct = 0
+    content_sim_total = 0
+    detailed_results = []
+    n = len(gold_data)
+    for gold, hunter in zip(gold_data, hunter_data):
```

Ruff (0.11.9)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)